calculate_trigger.py

```diff
@@ -26,15 +26,38 @@ def allign_fault_to_instruction(address, tbinfo_size, tbinfo_assembler, tbinfo_i
             return asm_addresses[i]
 
 
-def search_for_fault_location(
+def calculate_lifespan_from_start(
     filter_lists,
```

Comment:
Not sure, if I get this but `filter_lists` sounds misleading. Isn't it a simple instruction trace (like a list of ProgramCounter values)?

Reply:
Just kept the name like before. It is a list of list of all ins instructions inside the different tb executed. I am not sure if this PR is the scope to change this or it should be done in a different PR

Reply:
Nope, keep it at is.